spec/wings/active_fedora_converter_spec.rb

```diff
@@ -81,6 +81,10 @@
         it 'converts member_of_collection_ids back to af_object' do
           expect(converter.convert.members.map(&:id)).to match_array [work2.id, work3.id]
```

Comment:
If you want to demonstrate the difference between members and ordered_members, you can use match_array here and reverse the order of the ids...
```
          expect(converter.convert.members.map(&:id)).to match_array [work3.id, work2.id]
```

Reply:
👍